Src/FluentAssertions.DataSets/DataSetAssertions.cs

```diff
@@ -248,21 +247,7 @@ public AndConstraint<DataSetAssertions<TDataSet>> BeEquivalentTo(DataSet expecta
         var defaults = new DataEquivalencyAssertionOptions<DataSet>(AssertionOptions.CloneDefaults<DataSet>());
         IDataEquivalencyAssertionOptions<DataSet> options = config(defaults);
```

Comment:
Suggestion: change all three occurrences of `config(defaults)` to `_ = config(defaults);`.